js/main.js

```diff
@@ -286,3 +335,83 @@ function handlePlayAgain(event) {
   putLyrics(data.lyrics);
   lyricsSwap(data.lyricCard);
 }
+
+function handlePlaylist(event) {
+  event.preventDefault();
+  data.playlist.push({ id: data.playlistID, song: event.target.song.value, artist: event.target.artist.value });
+  $emptyPlaylistText.className = 'hidden center';
+  $ul.append(renderPlaylist(event.target.song.value, event.target.artist.value));
+  $form.reset();
+}
+
+function renderPlaylist(song, artist) {
+  var li = document.createElement('li');
+  var row = document.createElement('div');
+  var columnA = document.createElement('div');
+  var columnB = document.createElement('div');
+  var columnC = document.createElement('div');
+  var a = document.createElement('a');
+  var deleteIcon = document.createElement('i');
+  var p = document.createElement('p');
+  var br = document.createElement('br');
+  var button = document.createElement('button');
+
+  li.setAttribute('data-playlist-id', data.playlistID);
+  row.setAttribute('class', 'row');
+  columnA.setAttribute('class', 'column-a center');
+  columnB.setAttribute('class', 'column-b padding-top');
+  columnC.setAttribute('class', 'column-c center');
```

Comment:
Not a fan of these column-a, b, c names. what are these actually holding? In this case it would be good to use more descriptive class names.